src/zzz_od/application/commission_processing/commission_processing_app.py

```
@@ -0,0 +1,370 @@
+            clean_text = ''.join([c for c in ocr_result if c.isdigit() or c == '/'])
+            if '/' in clean_text:
+                parts = clean_text.split('/')
+                if len(parts) >= 2:
+                    current = int(parts[0])
+                    total = int(parts[1])
+                    if total > 0 and current >= total:
+                        return self.round_success('完成')
+                    # 只要第一个数字是1000也算完成
+                    if current >= 1000:
+                        return self.round_success('完成')
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)